lib/JunkDetection/CXX/CXXJunkDetector.cpp

```diff
@@ -17,26 +17,45 @@
 using namespace mull;
 
 template <typename Visitor>
-static bool isJunkMutation(ASTStorage &storage, MutationPoint *point) {
+static bool
+isJunkMutation(ASTStorage &storage, MutationPoint *point,
+               std::map<MutationPoint *, BeginEndPair> &mutationPointsRanges) {
```

Comment:
i'm curious if there is a better way to store/populate this information.
It would be nice if one can use the ASTStorage from a reporter and having access to the AST would be very helpful.

For a pretty long time I'm thinking how to implement something like `DiffReporter` that will report mutants as git patches.